script.js

```diff
@@ -0,0 +1,79 @@
+const Levels = document.querySelectorAll('.game-level');
+let level = 3;
+
+const difficultyLevel = (e) => {
+	Levels.forEach(item => item.classList.remove('active-level'));
+	e.target.classList.add('active-level');
+  	level = e.target.dataset.level;
+
+  	let three = document.querySelector('[data-level="3"]');
+
+  	if (e.target.dataset.level === undefined) {
+  		level = 3;
+  		three.classList.add('active-level');
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5876445)